scripts/docs/utils.js

```diff
@@ -139,3 +144,28 @@ function writeFile( filePath, data ) {
 function normalizePath( modulePath ) {
 	return modulePath.split( path.sep ).join( path.posix.sep );
 }
+
+/**
+ * Adds ts-loader with proper configuration to the passed configuration object
+ * only if the `tsconfig.test.json` file exists.
+ *
+ * @param {Object} webpackConfig
+ * @returns {Void}
+ */
+function addTypeScriptLoader( webpackConfig ) {
+	const tsconfigPath = path.resolve( process.cwd(), 'tsconfig.test.json' );
```

Comment:
Are you sure that this is the correct file? Building production snippets in docs should use `tsconfig.test.json` file?

Reply:
Production snippets will not use it (L158). I would rename this configuration from `test` to `dev`. It's a development configuration. Used by us when developing things and checking them by running tests or building docs.